providers/gcp/resources/discovery.go

```diff
@@ -35,8 +38,16 @@ const (
 	DiscoveryGkeClusters        = "gke-clusters"
 	DiscoveryStorageBuckets     = "storage-buckets"
 	DiscoveryBigQueryDatasets   = "bigquery-datasets"
+	DiscoverCloudSQLMySQL       = "cloud-sql-mysql"
+	DiscoverCloudSQLPostgres    = "cloud-sql-postgres"
```

Comment:
Let's use the full name. They've used that name for at least a decade. 

```suggestion
	DiscoverCloudSQLPostgreSQL    = "cloud-sql-postgresql"
```

Reply:
This is not what we get back from Google's API  
* https://github.com/googleapis/google-api-go-client/blob/main/sqladmin/v1beta4/sqladmin-gen.go#L1472
* https://github.com/googleapis/google-api-go-client/blob/main/sqladmin/v1beta4/sqladmin-gen.go#L534-L542

So we will have to do some string replacement.